ocaml/jane/doc/extensions/error_message.md

```diff
@@ -0,0 +1,38 @@
+# The `[@error_message]` attribute
+
+You can put the `[@error_message]` attribute on type or kind constraints
+to add custom text to error messages. This can be useful in the output
```

Comment:
Is this only intended for use in ppxes? If so it should say that explicitly

Reply:
I've connected the two sentences more -- see if you like.